targets/spidermonkey/templates/ifunction.c

```diff
@@ -39,7 +39,11 @@ bool ${signature_name}(JSContext *cx, uint32_t argc, jsval *vp)
                              "class_name": $class_name,
                              "level": 2,
                              "ntype": str($arg)})};
-            #set $arg_array += ["arg"+str(count)]
+            #if arg.is_pointer or arg.is_numeric
```

Comment:
Good callout. I was generating std::move calls for enums and booleans. This should be corrected now.